engine/graphics/src/opengl/graphics_opengl_private.h

```diff
@@ -41,7 +41,7 @@ namespace dmGraphics
     {
         TextureParams     m_Params;
         TextureType       m_Type;
-        GLuint*           m_TextureIds;
+        uint32_t*         m_TextureIds;
```

Comment:
Here and below - changed type from `GLuint` to `uint32_t`. Instead of storing GL handle directly now store index in `m_AllGLHandles`.